app/lib/submission_builder/ty2024/states/id/id_return_xml.rb

```diff
@@ -36,8 +36,8 @@ def supported_documents
                 include: true
               },
               {
-                xml: nil, # TODO 39RForm xml to be added in future story
-                pdf:  PdfFiller::Id39rPdf,
+                xml: SubmissionBuilder::Ty2024::States::Id::Documents::Id39r,
+                pdf: PdfFiller::Id39rPdf,
                 include: @submission.data_source.dependents.count > 4
               },
```

Comment:
i would make this include: true because the id39R should be included not just for listing dependents